### PR TITLE
TensorFlow fixes

### PR DESCRIPTION
**Pull request by sfilipi**
Closing #3481 by rearranging where the explanation lives for the TensorFlow transfomer/estimator, and fixing the links.

### CODE REVIEW

File: src/Microsoft.ML.TensorFlow/TensorflowTransform.cs [hunk 1]
```diff
@@ -1017,9 +1019,7 @@ public TFTensor GetBufferedBatchTensor()
         }
     }
 
-    /// <summary>
-    /// Estimator for the <see cref="TensorFlowTransformer"/>.
-    /// </summary>
+    /// <include file='doc.xml' path='doc/members/member[@name="TensorflowTransformer"]/*' />
```
Comment (abgoswam) [inline src/Microsoft.ML.TensorFlow/TensorflowTransform.cs]: >/// [](start = 4, length = 3)

issue #3481  says  the transform should suggest the package  it is part of .  

Has that been added somewhere in this PR ?

Comment (artidoro) [reply src/Microsoft.ML.TensorFlow/TensorflowTransform.cs]: As a follow up to @abgoswam, why are we not using the usual estimator doc template here?

File: src/Microsoft.ML.TensorFlow/TensorFlowModel.cs [hunk 2]
```diff
@@ -52,14 +52,14 @@ public DataViewSchema GetInputSchema()
         /// <summary>
         /// Scores a dataset using a pre-traiend <a href="https://www.tensorflow.org/">TensorFlow</a> model.
         /// </summary>
-        /// <param name="inputColumnName"> The name of the model input.</param>
-        /// <param name="outputColumnName">The name of the requested model output.</param>
+        /// <param name="inputColumnName"> The name of the model input. The data type is a vector of <see cref="System.Single"/>.</param>
```
Comment (zeahmed) [inline src/Microsoft.ML.TensorFlow/TensorFlowModel.cs]: ><see cref="System.Single"/> [](start = 101, length = 27)

Input can be of any  type. Not necessarily of float. Look at the following test
[https://github.com/dotnet/machinelearning/blob/f9f327ad314c2b175d9f676ef068a2ff1114c21d/test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs#L215](https://github.com/dotnet/machinelearning/blob/f9f327ad314c2b175d9f676ef068a2ff1114c21d/test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs#L215)

File: src/Microsoft.ML.TensorFlow/TensorflowTransform.cs (hunk 1 above)
Comment (sfilipi) [reply src/Microsoft.ML.TensorFlow/TensorflowTransform.cs]: I checked that the existing documentation states that. 

[https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.TensorFlow/doc.xml#L31](https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.TensorFlow/doc.xml#L31)

---
In reply to: [277842926](https://github.com/dotnet/machinelearning/pull/3537#discussion_r277842926) [](ancestors = 277842926)

File: src/Microsoft.ML.TensorFlow/TensorFlowModel.cs [hunk 3]
```diff
@@ -52,14 +52,14 @@ public DataViewSchema GetInputSchema()
...
+        /// <param name="outputColumnName">The name of the requested model output. The data type is a vector of <see cref="System.Single"/></param>
         /// <param name="addBatchDimensionInput">Add a batch dimension to the input e.g. input = [224, 224, 3] => [-1, 224, 224, 3].
         /// This parameter is used to deal with models that have unknown shape but the internal operators in the model require data to have batch dimension as well.</param>
         /// <example>
         /// <format type="text/markdown">
         /// <![CDATA[
-        /// [!code-csharp[ScoreTensorFlowModel](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlowTransform.cs)]
+        /// [!code-csharp[ScoreTensorFlowModel](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/ImageClassification.cs)]
```
[the remark this answers is not in the document]
Comment (sfilipi) [reply src/Microsoft.ML.TensorFlow/TensorFlowModel.cs]: I guess it'll get taken care during rebase. 

---
In reply to: [277846603](https://github.com/dotnet/machinelearning/pull/3537#discussion_r277846603) [](ancestors = 277846603)

File: src/Microsoft.ML.TensorFlow/TensorflowTransform.cs (hunk 1 above)
Comment (sfilipi) [reply src/Microsoft.ML.TensorFlow/TensorflowTransform.cs]: In the interest of time. All the information that the template establishes is already there. The file is referenced from the main package too. 

---
In reply to: [277847407](https://github.com/dotnet/machinelearning/pull/3537#discussion_r277847407) [](ancestors = 277847407)

File: src/Microsoft.ML.TensorFlow/TensorFlowModel.cs (hunk 3 above)
Comment (shmoradims) [inline src/Microsoft.ML.TensorFlow/TensorFlowModel.cs]: >ImageClassification [](start = 119, length = 19)

ZeeshanS also touched this line. please talk with him if merge conflicts happens.

**Comment by sfilipi** [comment]
close/reopen to trigger builds

**Comment by sfilipi** [comment]
close/reopen to trigger build

**Pull request merged**